lib/cronex/exp_descriptor.rb

```diff
@@ -129,19 +130,19 @@ def time_of_day_description(expression_parts)
       description = ''
       if [sec_exp, min_exp, hour_exp].all? { |exp| !Cronex::Utils.include_any?(exp, SPECIAL_CHARS) }
         # specific time of day (i.e. 10 14)
-        description += resources.get('at') + ' ' + Cronex::Utils.format_time(hour_exp, min_exp, sec_exp)
+        description += resources.get('at') + ' ' + Cronex::Utils.format_time(hour_exp, min_exp, sec_exp, @timezone)
```

Comment:
I'd add `:timezone` to the list of `attr_accessor` and use it w/o the `@` prefix in all the places